src/bindings/python/tests/test_runtime/test_sync_infer_request.py

```diff
@@ -88,15 +89,16 @@ def test_get_profiling_info(device):
     softmax = ops.softmax(param, 1, name="fc_out")
     model = Model([softmax], [param], "test_model")
 
-    core.set_property(device, {"PERF_COUNT": "YES"})
+    core.set_property(device, {ov.properties.enable_profiling: "YES"})
```

Comment:
@jiwaszki can we use `True` / `False` instead of string value `YES` / `NO` ?

Reply:
I changed the strings to their respective boolean values and ran pytest without encountering any errors. Additionally, in other parts of the code, True is used instead of "YES", reference: 

https://github.com/openvinotoolkit/openvino/blob/f9605cd8d4c488ad405397bc4294e35a403ae803/src/bindings/python/tests/test_runtime/test_properties.py#L524

Therefore, I believe this approach is consistent, and I have made these changes as well.